wagtailio/project_styleguide/templates/patterns/components/sign_up_form/base_form.html

```diff
@@ -0,0 +1,17 @@
+<div class="sign-up-form {% if classes %}{{ classes }}{% endif %}">
```

Comment:
Suggestion: Wrap this in a grid class so it isn't full width. The following css on `.sign-up-form` should work:

```
grid-column: 2 / span 2;
  
@include media-query(medium) {
  grid-column: 2 / span 3;
}

@include media-query(large) {
  grid-column: 2 / span 5;
}
```

Reply:
Good point thanks!